src/main/java/net/mcreator/ui/modgui/PotionGUI.java

```diff
@@ -132,6 +140,10 @@ public PotionGUI(MCreator mcreator, @Nonnull ModElement modElement, boolean edit
 		}
 	}
 
+	@Override protected List<JEntriesList> getEntryLists() {
```

Comment:
Preview generator uses different system to do this, use the same system as for other elements please